lms/services/moodle.py

```diff
@@ -137,14 +137,15 @@ def list_files(self, course_id: int):
     def file_exists(self, file_id) -> bool:
         """Check if the file exists in the course."""
         # Moodle file IDs are URLs, but they need the token to be accessible
-        response = self._http.request("HEAD", f"{file_id}&token={self.token}")
+        response = self._http.request(
+            "GET", f"{file_id}&token={self.token}", headers={"Range": "bytes=0-5"}
```

Comment:
I had to google the `Range` header. Leaving the docs here for future readers context https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range